purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesFactory.kt

```diff
@@ -50,6 +51,7 @@ import java.util.concurrent.Executors
 import java.util.concurrent.ThreadFactory
 
 internal class PurchasesFactory(
+    private val isDebugBuild: IsDebugBuildProvider,
```

Comment:
Hmm by the name `isDebugBuild` I would think this to be a boolean... Maybe we should rename to `debugBuildProvider` and have a `get()` method or something instead?

Reply:
Hmm, I see what you mean, but the reason is that we call it as a function, on line 79 below: 
```kotlin
isDebugBuild()
```  
Imo that's "worth it", because otherwise that line would become something like: 
```kolin
debugBuildProvider.get()
``` 
which I think is less readable. 

What do you think?

Reply:
Yeah makes sense, while there might be some confusion when reading the name of the property, it's more legible after so I'm fine with it 👍